parlai/agents/image_seq2seq/modules.py

```diff
@@ -251,6 +250,16 @@ def encode_images(
                 )
             )
             assert image_masks.shape == image_encoded.shape[:2]
+        else:
+            image_encoded = torch.stack(
+                [self.dummy_image_enc for _ in range(len(images))]
+            ).reshape(
+                len(images),
+                self.n_image_tokens * self.n_image_channels,
+                self.embedding_size,
+            )
+            image_masks = torch.stack([~self.ones_mask for _ in range(len(images))])
+            assert image_masks.shape == image_encoded.shape[:2]
```

Comment:
Hmm we can move this line out of the `else` block because it's also applied in the `if` block